From 90dbed0ca2abd632649078b7a8b0308977d54fae Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:08 -0400
Subject: [tools] perf/tests: add a test of kvm-390 trace event

Message-id: <1410181752-25631-184-git-send-email-jolsa@redhat.com>
Patchwork-id: 91867
O-Subject: [PATCH RHEL7.1 BZ1134356 183/307] perf tests: Add a test of kvm-390: trace event
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit c0bc8c6d4a2e18e54ef7854a86bc5a47a8e3d04e
Author: Alexander Yarygin <yarygin@linux.vnet.ibm.com>
Date: Fri Apr 25 17:34:07 2014 +0200

description
===========
Add a s390 specific test of a hardcoded trace event with '-'
in the name.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/tests/parse-events.c b/tools/perf/tests/parse-events.c
index 81dbd5a..deba669 100644
--- a/tools/perf/tests/parse-events.c
+++ b/tools/perf/tests/parse-events.c
@@ -1389,6 +1389,13 @@ static struct evlist_test test__events[] = {
   .check = test__pinned_group,
   .id    = 41,
  },
+#if defined(__s390x__)
+ {
+  .name  = "kvm-s390:kvm_s390_create_vm",
+  .check = test__checkevent_tracepoint,
+  .id    = 100,
+ },
+#endif
 };
 
 static struct evlist_test test__events_pmu[] = {
-- 
1.7.1